docs/source/build-run-qualcomm-ai-engine-direct-backend.md

```diff
@@ -134,12 +138,12 @@ Commands to build `qnn_executor_runner` for Android:
 
 ```bash
 cd $EXECUTORCH_ROOT
-mkdir build_android
-cd build_android
+mkdir cmake_android_out
+cd cmake_android_out
```

Comment:
why this change is needed?

Reply:
can use old one. simply it's already in gitignore.

Reply:
let's use the old one probably? Try to make the change minimum

Reply:
Done